OptionSelector/OSCategoriesTableViewController.h

```diff
@@ -0,0 +1,13 @@
+//
+//  OSCategoriesTableViewController.h
+//  OptionSelector
+//
+//  Created by Daniel Distant on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface OSCategoriesTableViewController : UITableViewController
```

Comment:
Style: When naming classes, it's best to use the same prefix for all of the classes in your app. For example, since you started out with `CQCategory`, this one should probably be `CQCategoriesTableViewController`. This is important because when I read through the rest of your code, it helps me figure out where each class is defined. The prefix you're using here, `OS`, is actually used by some of Apple's C APIs.